R/wrangle_and_check.R

```diff
@@ -80,13 +80,8 @@ wrangle_and_check_pacta_results <- function(pacta_results, start_year, time_hori
                                             scenario_geography_filter, scenarios_filter,
                                             equity_market_filter) {
   wrangled_pacta_results <- pacta_results %>%
-    dplyr::filter(!is.na(.data$scenario)) %>%
-    check_scenario_settings(scenario_selections = scenarios_lookup) %>%
-    dplyr::filter(.data$scenario %in% scenarios_lookup) %>%
-    # TODO: temporary fix, remove once all scenario data is used from scenario file
-    dplyr::filter(!(.data$scenario == "ETP2017_NPS" & .data$ald_sector == "Power")) %>%
+    select_sector_scenario_combinations() %>%
     dplyr::mutate(scenario = sub(".*?_", "", scenario)) %>%
-    dplyr::mutate(scenario = stringr::str_replace(.data$scenario, "NPSRTS", "NPS")) %>%
```

Comment:
this is finally removed? I thought I wouldn't live to see the day

Reply:
Don`t scare me, but I found no reason to leave it here. Afterall we filter for scenarios before and the is no change for an npsrt to make it through.  For this line to make sense it would have to happen before the filtering..

Reply:
🥃